masschallenge/django-accelerator
# AC-8362 Reflect more inclusive options for gender

### Changes Introduced in [AC-8362 ](https://masschallenge.atlassian.net/browse/AC-8362 ):
- A schema migration for the gender object
- A data migration to add the options provided by the business
- Updated factories
### Testing:
- Check out this branch
- On impact, run `docker-compose run --rm web ./manage.py migrate accelerator`
- Navigate to one of the profiles on the admin page e.g(this [profile](http://localhost:8181/admin/mc/expertprofile/79/change/))
- Notice the gender fields
- Confirm that the changes are consistent with the ticket description
